src/app/controllers/revGraphView.js

```diff
@@ -0,0 +1,1044 @@
+              d.fy = d3.event.y;
+              }
+              function dragended(d) {
+              if (!d3.event.active) simulation.alphaTarget(0);
+              d.fx = null;
+              d.fy = null;
+              }
+              return d3.drag()
+                  .on("start", dragstarted)
+                  .on("drag", dragged)
+                  .on("end", dragended);
+            }
+
+            var svg_node = function(d) {
+              console.log('called svg node function on nodes: ', d)
+              var result = d.append("g")
+              .attr("stroke", colorByGroup)
+              .attr("fill", colorByGroup);
+          
+              var svg = result.append("use")
+                  .attr("xlink:href", calcSymbolId)
+                  .attr("width", 20)
+                  .attr("height", 20)
+                  .attr("transform", d => {
+                      let selectedFactor = (d.id == selectedNodeId) ? 2.0 : 1.0;
+                      let scale = (d.nodeScale || 1.0) * selectedFactor;
+                      return "scale(" + scale  + ")"
+                  })
+                  .attr("style", d => "opacity:" + (d.opacity || 0.8));
+
+              svg.on("click", d => {
+                  console.log("clicked on ", d);
+                  var selectedNodeId = d.id;
+                  svg.attr("transform", d => { //recalc scale
+                      let selectedFactor = (d.id == selectedNodeId) ? 2.0 : 1.0;
+                      let scale = (d.nodeScale || 1.0) * selectedFactor;
+                      return "scale(" + scale  + ")"
+                  })
+                  var selectedNode = d.__proto__
+                          d3.select("#selectedNode_" + scope.data.mainNode)
+                          .html("<p>" + node_as_html_table(selectedNode) + "</p>");
+                          if (selectedNode.rel == "itemReviewed") {
+                          }
+                          if (selectedNode['@type'].endsWith("Review")) {
+                              d3.select("#reviewNode_" + scope.data.mainNode)
+                              .html(`
+                                     <br>
+                                     <span class="rate" title="This review is accurate (help us improve our AI)" id="accRev">
+                                     <a class="icon-ok"></a>
+                                     <span class="accurate-stat" title="Number of Co-inform users who have rate this review as accurate"></span>
+                                     </span>
+                                     |
+                                     <span title="This review is inaccurate (help us improve our AI)" id="inaccRev">
+                                     <a class="icon-remove"></a>
+                                     <span class="accurate-stat" title="Number of Co-inform users who have rate this review as inaccurate"></span>
+                                     </span>
+                                    `
```

Comment:
Since we should change the table vision with the "credibility card" style, it is better work directly on that new part of the code instead of this one.